rfc_bibtex/rfc_bibtex.py

```diff
@@ -35,8 +36,32 @@ def __init__(self, id_names=[], in_file_name=None, out_file_name=None):
         if in_file_name is not None:
             self._id_names += self._read_ids_from_file(in_file_name)
 
+    @staticmethod
+    def _rfc_key_function(name):
+        """Turn RFC32 into RFC00032 for sorting"""
+        try:
+            if name.startswith('RFC'):
+                return "RFC"+format( int(name[3:]), "05d")
+        except (ValueError,TypeError) as e:
+            pass
```

Comment:
This should be handled properly (*e.g.* a custom exception that's propagated)

Reply:
Okay, but what's the advantage of using a custom exception here?

Reply:
> Okay, but what's the advantage of using a custom exception here?

Well, generally speaking, it's a good `OO` practice: you're separating Python errors (*e.g.* ValueError), from your application errors (*e.g.* NotAStringException). But in this case, however, you are right. The application is very simple, so there is no need to over-complicate at this point. Maybe just raising a `ValueError` with a message something in the lines of: "Expecting RFCXXXX, but instead got {name}" (change the wording as you wish)? Alternatively, you can just remove the exception handling block altogether.

Is there any particular reason you added that 'try/except`?

Reply:
The try/except was added because a \cite{RFCabc} might have resulted with an error on the `int()` cast. Although such a label might not have made it through because of the regular expression. Still, I've been burned by that in the past, so I just wanted to cover all bases.

Reply:
("cover all bases" is an analogy and a reference to the American game of Baseball.  See https://idioms.thefreedictionary.com/cover+all+the+bases)

Reply:
> The try/except was added because a \cite{RFCabc} might have resulted with an error on the `int()` cast. Although such a label might not have made it through because of the regular expression. Still, I've been burned by that in the past, so I just wanted to cover all bases.

Okay, makes sense. I'm thinking maybe it's a good idea to log a warning message, when the program is executed in verbose mode (`-v` switch)? Just so that the user is not left wondering why something was not processed. What do you think? But we can do this a separate pull request (for the addition of the `-v` switch).